 
[-07-2025 07-40] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C^C 
[-07-2025 07-40] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C^C 
[-07-2025 07-41] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C^C 
[2025 -07-16 07-42] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C^C 
[2025 -07-16 08-21] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C 
[2025 -07-16 08-22] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C 
[2025 -07-16 08-22] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C 
[2025 -07-16 08-23] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C[2025 -07-16 08-26]  
[2025 -07-16 08-26]  
[2025 -07-16 08-27]  
[2025 -07-16 08-23] !MSG! 
[2025 -07-16 08-27]  
[2025 -07-16 08-27]  
[2025 -07-16 08-27]  
 
[2025 -07-16 08-28] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
^C^C 
[2025 -07-16 08-31] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
[2025 -07-16 08-31] Reconectando em 5 segundos... 
 
[2025 -07-16 09-04] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
C:\Users\pedro\AppData\Roaming\npm\node_modules\localtunnel\bin\lt.js:81
    throw err;
    ^

Error: connection refused: localtunnel.me:33841 (check your firewall settings)
    at Socket.<anonymous> (C:\Users\pedro\AppData\Roaming\npm\node_modules\localtunnel\lib\TunnelCluster.js:52:11)
    at Socket.emit (node:events:518:28)
    at emitErrorNT (node:internal/streams/destroy:170:8)
    at emitErrorCloseNT (node:internal/streams/destroy:129:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)

Node.js v22.17.0
[2025 -07-16 09-04] Erro detectado. C�digo de erro: 1 
[2025 -07-16 09-04] Reconectando em 5 segundos... 
 
[2025 -07-16 09-56] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
C:\Users\pedro\AppData\Roaming\npm\node_modules\localtunnel\bin\lt.js:81
    throw err;
    ^

Error: connection refused: localtunnel.me:18219 (check your firewall settings)
    at Socket.<anonymous> (C:\Users\pedro\AppData\Roaming\npm\node_modules\localtunnel\lib\TunnelCluster.js:52:11)
    at Socket.emit (node:events:518:28)
    at emitErrorNT (node:internal/streams/destroy:170:8)
    at emitErrorCloseNT (node:internal/streams/destroy:129:3)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)

Node.js v22.17.0
[2025 -07-16 09-56] Erro detectado. C�digo de erro: 1 
[2025 -07-16 09-56] Reconectando em 5 segundos... 
 
[2025 -07-16 12-26] Tentando iniciar tunnel em https://ativarestoque3.loca.lt 
your url is: https://ativarestoque3.loca.lt
